example/src/main/java/org/eclipse/hono/vertx/example/base/HonoExampleConstants.java

```diff
@@ -0,0 +1,46 @@
+package org.eclipse.hono.vertx.example.base;
+
+/**
+ * Class defines where to reach Hono's microservices that need to be accessed for sending and consuming data.
+ * This is intentionally done as pure Java constants to provide an example with minimal dependencies (no Spring is
+ * used e.g.).
+ *
+ * Please adopt the values to your needs - the defaults serve for a typical docker swarm setup.
+ * TODO: check and (possibly) change to kubernetes values.
+ */
+public class HonoExampleConstants {
+    /**
+     Define the host where Hono's microservices can be reached.
+     */
+    public static final String HONO_CONTAINER_HOST = "192.168.99.100";
```

Comment:
I think the default should be 127.0.0.1 so that at least those folks do not need to change it who do not use Docker Machine. Those who do will need to adapt this anyway ...

Reply:
Good point. Will adopt this.